lib/themes/dosomething/paraneue_dosomething/js/components/InstructionSlide.js

```diff
@@ -28,7 +40,7 @@ class InstructionSlide extends React.Component {
             <div className="container__row">
               <div className="container__block -narrow -centered">
                 <div className="media-video">
-                  <video poster="https://static.dosomething.org/onboarding/poster.png" src="https://static.dosomething.org/onboarding/CouldBeYou_720p.mp4" loop autoPlay controls width="100%"></video>
+                  <video id="onboarding-video" poster="https://static.dosomething.org/onboarding/poster.png" src="https://static.dosomething.org/onboarding/CouldBeYou_720p.mp4" loop  autoPlay={this.props.isActive ? 'true' : 'false'} controls width="100%"></video>
```

Comment:
Can you update this to use [a `ref` attribute](https://facebook.github.io/react/docs/more-about-refs.html#the-ref-callback-attribute)? Hard-coding an `id` and querying the document for it means this would break if we ever had two on a page. (Unlikely, I know but a good habit to keep.)